manager/volume.go

```diff
@@ -166,6 +168,16 @@ func (m *VolumeManager) Create(name string, spec *types.VolumeSpec) (v *longhorn
 			return nil, fmt.Errorf("cannot get backup %v: %v", spec.FromBackup, err)
 		}
 		logrus.Infof("Override size of volume %v to %v because it's from backup", name, backup.VolumeSize)
+
+		// If KubernetesStatus is set on Backup, restore it.
+		if statusJSON, ok := backup.Labels[types.KubernetesStatusLabel]; ok {
+			if err := json.Unmarshal([]byte(statusJSON), kubeStatus); err != nil {
+				return nil, err
```

Comment:
Don't error out. Print an error log then proceed. It's not critical enough to stop restoration.